### PR TITLE
combine cetaceans :whale2: and fish and invertebrates :fish: sections…

### PR DESCRIPTION
**Pull request by stoyelq**
Combines these two sections into a single large section.   #68

@gomezcatalina does the text at the start of the cetaceans section need to modified / where should this go?  

@stfinnis: does any of the OBIS section need to be updated because it's now just one section?  I did update the table and figure captions, but I may have missed things.

### CODE REVIEW

File: app/templates/EN/species/species.Rmd [hunk 1]
```diff
@@ -3,6 +3,33 @@ output: html_document
...
...helf bioregion are opportunistic; comprehensive systematic surveys on the occurrence or distribution of cetaceans across the entire region are limited. Results in this report provide some information on distribution but should not be interpreted as comprehensive indices of population abundance or relative density or abundance within the study area, as the related effort does not cover the entirety of a species' range and is unequal in coverage across the study area. A future comprehensive analysis of any associated observational effort for these data is necessary to further characterize cetacean presence in the area over temporal and spatial scales. For surveys with either random placement of transects within the study area, or surveys with estimates of distance to sighting, density and distribution of species could be estimated (though such study results are not presented here). Consequently, outputs of this report include presence-only data, and we cannot say whether a lack of presence of a species relates to a real absence, a result of lack of effort, or because additional data sources were not reported for the area (e.g., sightings data from other sources, acoustic detections).
```
Comment (stoyelq) [inline app/templates/EN/species/species.Rmd:35]: Does this block of text need to be updated? @gomezcatalina

**Review by stfinnis — CHANGES_REQUESTED** [review]
From what I can tell, you haven't missed anything from merging the OBIS/GBIF fish & inverts/cetacean sections. Thank you for doing this! I'll let Catalina comment about the overall Cetaceans text.
I just provided to comments about updating the OBIS/GBIF information in the obisCommon_en.Rmd so it has current (April 2022) information.
Probably the OBIS/GBIF references in the bib file and in-text citations should also be changed from 2021 --> 2022. I'm still not totally sure how pull requests work. Do I just identify the problem here, or should I actually make these changes (here? or in a separate pull request?) so you don't have to?

**Review by stfinnis — APPROVED** [review]

File: app/templates/EN/species/species.Rmd (hunk 1 above)
Comment (gomezcatalina) [reply app/templates/EN/species/species.Rmd:35]: mmm... maybe a bit... particularly where to place that text exactly. We may need to arrange things (ToC) a bit and update headers? And the wording between modules and sections? :confused: At the moment, we have main headers for modules, which are the items that can be selected via the Shinny App e.g. "Species", "Aquatic Invasive Species", "Habitat"... We have secondary headers for sections, which correspond to information summarized in the modules. By merging cetaceans and fish/inverts, we have a whole section specifically for cetaceans... but we do not have something like this for fish/invertebrates. What if we create a section, within the new Species module, so the table of contents will look like:

**SPECIES MODULE**

_INFORMATION FROM THE NATIONAL AQUATIC SPECIES AT RISK GEODATABASE_

_DATASETS FOR MULTIPLE SPECIES_

- obis
- ...

_FISH AND INVERTEBRATES DATASETS_

- RV Survey...

_CETACEAN DATASETS_

- wsdb...

**Review by gomezcatalina — CHANGES_REQUESTED** [review]
Proposing the creation of a slightly different Table of Contents to accommodate our brand new amalgamated "Species" tab. We may need @stfinnis :mage: magic powers to advice and word smith. I'll create a meeting to explore options.

**Review by gomezcatalina — APPROVED** [review]
Merge of section with new updates ToC looks great :star2:

**Pull request merged**